models/alignn_ff/alignn_ff_relax.py

```diff
@@ -0,0 +1,108 @@
+# %%
+from __future__ import annotations
+
+import os
+
+import numpy as np
+import pandas as pd
+from pqdm.processes import pqdm
```

Comment:
Can we work around this new dependency? I'm not sure how it differs from `tqdm`? Is `tqdm` lacking parallel process support?

Reply:
Yes, you can just use tqdm. For parallel processing pqdm has a bit nicer output, that's all.